tests/cypress/support/commands.ts

```diff
@@ -709,3 +715,38 @@ Cypress.Commands.add('compareClusterResourceCount', (clusterName) => {
     })
   })
 })
+
```

Comment:
Suggestion: add TODO: remove when common ecp library is updated

Reply:
I will add it, next patchset.

Reply:
Added it.